spec/models/user_spec.rb

```diff
@@ -8,7 +8,7 @@
   it { should have_secure_password }
   it { should validate_length_of :state }
   it { should validate_length_of :zip_code }
-  it { should have_many :orders }
+  xit { should have_many :applications } #need to update orders
```

Comment:
@roscalabrin Same as comment above. If you already have a story for applications, make a comment in that card. It'll be easier to make sure this gets done.